task/buildah-pr/0.1/buildah-pr.yaml

```diff
@@ -51,19 +51,18 @@ spec:
       volumeMounts:
         - name: varlibcontainers
           mountPath: /var/lib/containers
-      securityContext:
-        privileged: true
-
+      env:
+      - name: STORAGE_DRIVER
+        value: vfs
     - name: push
       image: $(params.BUILDER_IMAGE)
       workingDir: /workspace/source
       command: ['buildah', 'push', '--tls-verify=$(params.TLSVERIFY)', '$(resources.outputs.image.url)', 'docker://$(resources.outputs.image.url)']
       volumeMounts:
         - name: varlibcontainers
           mountPath: /var/lib/containers
-      securityContext:
-        privileged: true
+      env:
+      - name: STORAGE_DRIVER
+        value: vfs
 
-  volumes:
```

Comment:
Should we remove this?

Reply:
no we still need this

Reply:
oh way, what did I remove that :sweat: will fix

Reply:
fixed